### PR TITLE
Import network whitelists with the sync_networks command

### PR DESCRIPTION
**Pull request by mark-ignacio**
Implements #938 - I added the `--authoritative` option to allow folks to keep the whitelist tidy.

Tested on a Python 2.7 deployment.

### CODE REVIEW

**Comment by mikegrima** [comment]
Could you add in some unit tests?

**Comment by mark-ignacio** [comment]
Oof, setting up a test environment is going to be some effort. Do you mind if I throw commits at this PR to get Travis to run them?

**Comment by mikegrima** [comment]
Absolutely. I know that it's a pain, however, we have been burnt many, many times by insufficient tests.

**Comment by mikegrima** [comment]
You could follow this as a good starting point: https://github.com/Netflix/security_monkey/blob/develop/security_monkey/tests/utilities/test_s3_canonical.py

File: security_monkey/manage.py [hunk 1]
```diff
@@ -681,6 +682,46 @@ def sync_swag(owner, bucket_name, bucket_prefix, bucket_region, account_type, sp
     app.logger.info('SWAG sync successful.')
 
 
+@manager.option('-b', '--bucket-name', dest='bucket_name', type=unicode, help="S3 bucket where network whitelist data is stored.")
+@manager.option('-i', '--input-filename', dest='input_filename', type=unicode, default='networks.json', help="File path or bucket prefix to fetch account data from. Default: networks.json")
+@manager.option('-a', '--authoritative', dest='authoritative', default=False, action='store_true', help='Remove all networks not named in `input_filename`.')
+def sync_networks(bucket_name, input_filename, authoritative):
+    """Imports a JSON file of networks to the Security Monkey whitelist."""
+    if bucket_name:
```
Comment (mikegrima) [inline security_monkey/manage.py]: Can we get a test with S3 syncing?

**Pull request merged**